docs/install/install_everest_helm_charts.md

```diff
@@ -0,0 +1,112 @@
+# Install Percona Everest using Helm
+
+Helm is a package manager for Kubernetes applications.
+
+The Percona Everest Helm chart can be set up with a single command and includes all the resources required to deploy and manage Percona Everest on your Kubernetes cluster. 
+
+The Helm chart simplifies the Percona Everest deployment process by packaging the configuration, dependencies, and templates. This streamlined approach ensures that everything can be set up smoothly without any hassle. For more information on Helm, refer to the official [documentation](https://helm.sh/docs/topics/charts/){:target="_blank"}.
+
+## Install Percona Everest
+
+You can install Percona Everest using Helm as an alternative method.
+{.power-number}
```

Comment:
I know we already spoke about this a few weeks back, but I still think a lot of this information is redundant, or not required. It makes it sound like the listed benefits are unique to the Everest Helm chart, but in reality is just a standard thing offered by Helm 😅 

I would go with something more concise..

 
```suggestion
# Install Percona Everest using Helm
This guide demonstrates how to install Percona Everest using [Helm](https://helm.sh/) as an alternative to `everestctl install`. Helm charts simplify deployment by packaging all required resources and configurations, making it an ideal choice for automating and managing installations in Kubernetes environments.

For more info, refer to the official documentation on [Helm charts](https://helm.sh/docs/topics/charts/#helm).

## Install Percona Everest
```

As reference, I'll attach how other cloud-native projects document their Helm install guides:
- https://linkerd.io/2-edge/tasks/install-helm/
- https://docs.cilium.io/en/stable/installation/k8s-install-helm/
- https://docs.gitlab.com/charts/installation/
- https://falco.org/docs/setup/kubernetes/

.. as you can see, the info and benefits about helm itself is deferred to the official docs, and these only focus on how to use the chart